src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -299,6 +299,54 @@ public virtual async Task PatchOne_Update_Test()
                 );
         }
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a computed field is missing from the request body.
+        /// </summary>
+        [TestMethod]
+        public virtual async Task PatchOneWithComputedFieldMissingFromRequestBody()
+        {
+            // Validate successful execution of a PATCH update when a computed field (here 'last_sold_on_date')
+            // is missing from the request body. Successful execution of the PATCH request confirms that we did not
+            // attempt to NULL out the 'last_sold_on_update' field.
+            string requestBody = @"
+            {
+                ""book_name"": ""New book"",
+                ""copies_sold"": 50
+            }";
+            string expectedLocationHeader = $"id/1";
+
+            await SetupAndRunRestApiTest(
+                    primaryKeyRoute: expectedLocationHeader,
+                    queryString: null,
+                    entityNameOrPath: _entityWithReadOnlyFields,
+                    sqlQuery: GetQuery("PatchOneUpdateWithComputedFieldMissingFromRequestBody"),
```

Comment:
`PatchOneUpdateWithComputedFieldMissingFromRequestBody` should be updated accordingly to validate that computed field wasn't nulled out.

Reply:
https://github.com/Azure/data-api-builder/pull/1596#discussion_r1322397055

Reply:
This `PatchOneUpdateWithComputedFieldMissingFromRequestBody` query would now need the NULL check that you added for other tests on the field: `last_sold_on_update`